docusaurus/docs/Android/03-ui/04-components/03-message-list.md

```diff
@@ -432,3 +432,16 @@ messageListView.setEmptyStateView(
 This code will display the following empty state:
 
 ![The custom empty state in action](../../assets/message_lis_custom_empty_state.png)
+## Configure When Avatar Appears
+Is it possible to configure when the avatar for messages appears. You can use `MessageListView.setShowAvatarPredicate` and pass a predicate to define when the avatar is going to be shown. The example implements the default behaviour:
+
+```
+messageListView.setShowAvatarPredicate(
+    ShowAvatarPredicate { messageItem ->
+        messageItem.positions.contains(MessageListItem.Position.BOTTOM) && messageItem.isTheirs
+    }
+)
+```
+:::note
+To avoid overlap between the avatar and the messages of the chat, remember to use `streamUiMessageStartMargin` and `streamUiMessageEndMargin` to create space for the avatar of the messages.
+:::
```

Comment:
We should show a concrete example here of how these attributes can be used. For example, based on these docs, I have no idea what value to set the margin to if I start showing avatars for my own messages so that they show up correctly.

It'd be nice to have something like

```
If you set a predicate that shows avatars for your own messages as well, use this value:

streamUiMessageEndMargin="???"

If your predicate doesn't show avatars for your own messages (this is the default behavior), remove the end margin:

streamUiMessageEndMargin="0"
```